### DISCUSSION TITLE
Do we have a visualization tool that can show the tree of a file? Can verify the query statement, which is really difficult as a beginner. Or more example for different language.

### DISCUSSION THREAD

**Discussion by LY1806620741** [tree-sitter/tree-sitter#3821] [Q&A] [answered]
### Problem

Do we have a visualization tool that can show the tree of a file? Can verify the query statement, which is really difficult as a beginner. Or more example for different language.

**Comment by sogaiu** [comment, marked as answer]
Is [this](https://tree-sitter.github.io/tree-sitter/playground) close to what you have in mind?
